submissions/emmanuelEsuong/css/styles-mediaQuery.css

```diff
@@ -0,0 +1,261 @@
+/* media-query (tablet)*/
+@media screen and (min-width: 768px) and (max-width: 900px) {
+        /* signUp page */
+    /* formfield */
+    #s-form {
+        border-radius: 25px;
+        padding: 35px 20px 20px 20px;
+        border: 2px 2px 2px 2px;
+    }
+    /* signupform */
+    #form {
+        overflow: hidden;
+        margin-top: 10px;
+        float: right;
+        margin-right: 215px;
+    }
+    #form input {
+        width: 550px;
+        padding: 10px;
+        border-radius: 7px;
+        border: solid 1px;
+    }
+    #table, #form {
+        display: inline-block;
+    }
+    #form #terms {
+        width: 50px;
+        float: left;
+        margin-left: 78px;
+    }
+    #termsLabel {
+        color: rgb(2, 209, 2);
+        font-weight: bold;
+    }
+    button {
+        border-radius: 3px;
+        border: 0px;
+        margin-left: 220px;
+        padding: 7px;
+        padding-left: 20px;
+        padding-right: 20px;
+        background-color: rgb(2, 209, 2);
+        font-weight: bold;
+        color: white;
+    }  
+    ul {
+        list-style-type: none;
+        padding: 0px;
+        overflow: hidden;
+    }
+    /* registration info */
+    #reg-acct {
+        font-size: 40px;
+        font-weight: bold;
+        color: rgb(2, 209, 2);
+    }
+    #heading, #reg-note, #acct-signin {
+        text-align: center;
+        margin-left: 295px;
+        padding-bottom: 0px;
+        padding-top: 0px;
+        font-size: 28px;
+        text-align: center;
+    }
+    /*signin instead*/
+    #signin{
+        color:rgb(2, 209, 2);
+    }
+    /*registration info container*/
+    #supT {
+        height: 880px;
+    }
+    /* footer */
+    footer {
+        height: 205px;
+    }
+    #icons {
+        margin-right: 60px;
+    }
+    #copyright {
+        margin-left: 235px;
+        margin-top: 40px;
+    }
+}
+/* media-query (mobile)*/
+@media screen and (max-width: 500px) {
+    /* top navigation */
+    .topnav {
+        overflow: hidden;
+        background-color: rgb(2, 209, 2);
+        text-align: center;
+        font-size: 35px;
+    }
+    .topnav #logo {
+        margin-left: 25px;
```

Comment:
If you are setting more than 2 properties, write it in shorthand. Set the others to 0